docs/oak_functions_abi.md

```diff
@@ -139,3 +139,44 @@ the value in that buffer, and returns it as part of `value_ptr_ptr` and
 
 - `result[0]: i32`: Status of operation as
   [`OakStatus`](https://github.com/project-oak/oak/blob/main/oak_functions/proto/abi.proto)
+
+### `tf_model_get_shape`
```

Comment:
I'm not sure why this is needed, isn't the shape part of the model? The caller should know what the shape is anyways.